packages/flutter/lib/src/widgets/banner.dart

```diff
@@ -19,6 +19,10 @@ const double _kOffset = 40.0; // distance to bottom of banner, at a 45 degree an
 const double _kHeight = 12.0; // height of banner
 const double _kBottomOffset = _kOffset + 0.707 * _kHeight; // offset plus sqrt(2)/2 * banner height
```

Comment:
While we're here, might as well fix this guy!

```suggestion
const double _kBottomOffset = _kOffset + math.sqrt1_2 * _kHeight;
```

Reply:
Changed to the square root constant from math